internal/pkg/aws/ecs/ecs.go

```diff
@@ -342,9 +345,24 @@ func (e *ECS) ActiveClusters(arns ...string) ([]string, error) {
 }
 
 // ActiveServices returns the subset of service arns that have an ACTIVE status.
-func (e *ECS) ActiveServices(clusterARN string, serviceARNs ...string) ([]string, error) {
+// Note that all services should be in the same cluster.
+func (e *ECS) ActiveServices(serviceARNs ...string) ([]string, error) {
+	if len(serviceARNs) == 0 {
```

Comment:
not sure if the api catches this error or just ignores it, but the max is 10

Reply:
i kinda think - if the ECS api catches these errors (0 ARNs specified, > 10 specified), we shouldn't explicitly handle them here. at least for 0, it seems like we already have logic to catch that error in the caller.

if the ECS api has some weird behavior on them - like returning all services in the cluster or something - then we can handle them explicitly.